### ISSUE TITLE
Newer versions of Ansible(2.17+) doesn't work on the geerlingguy/rockylinux8 box

### ISSUE THREAD

**Issue by mateusfmcota** [geerlingguy/ansible-for-devops#589]
The geerlingguy/rockylinux8 uses python 3.6 and according to [Ansible 2.17 Roadmap](https://docs.ansible.com/ansible/latest/roadmap/ROADMAP_2_17.html) they dropped support for it, this breaks all playbooks.

The way I fixed mine was to replace the box from the playbooks for the "rockylinux/9" one, which uses python 3.9 but the book probably needs to be updated

**Comment by geerlingguy** [comment]
Indeed... some of the examples will also need tweaking to work with `rockylinux9` too :(

I will be working on some book updates soon-ish, but can't promise any definite timeline for it!

**Comment by lethedata** [comment]
There are issues with running `sudo npx` on `rockylinux9` : https://github.com/geerlingguy/ansible-for-devops/pull/573#issuecomment-3599907359
